index.js

```diff
@@ -37,19 +38,32 @@ module.exports.iMeanMost =
 module.exports.butWhatActuallyIsCinnamonToastCrunchThough =
   function nobodyEvahExplainedItToMe() {
     return "Scientists still aren't sure, but we have a suspicion that it " +
-      "just might be a cereal involving cinnamon, toast, and something " +
-      "crunchy!"
+      "just might be something edible involving cinnamon, toast, and something " +
+      "umm... crunchy!"
   }
 
 module.exports.butWhyIsCinnamonToastCrunchGood = 
   function nobodyEvahExplainedThisToMeEither () {
-    return "It's simply a fact of life, you know? Birds fly, gravity falls, " +
-      "stevens universe, Cinnamon Toast Crunch is good."
+    return "It's simply a fact of life, you know? Birds fly, dogs walk, " +
+      "giraffes are tall, Cinnamon Toast Crunch is good."
   }
 
 module.exports.butCanIHaveItForDinner = 
-  function nobodyToldMeYouThatYouCould() {
+  function nobodyToldMeThatYouCouldnt() {
```

Comment:
@towerofnix Made a mistake with the title... Change it! Its on NPM!!!

Reply:
Oh that's kinda funny LMAO

Can you please file a separate pull request including just this change, if you have the moment. It's not related to milk, but this should totally be fixed, yes.